src/components/header/index.tsx

```diff
@@ -1,25 +1,39 @@
 import { ReactComponent as ProfileIcon } from '../../assets/profile.svg';
+import { ROUTE_PATH } from '../../routes';
 import { ReactComponent as RingIcon } from '../../assets/ring.svg';
 import { ReactComponent as SearchIcon } from '../../assets/search.svg';
 import { links } from '../navBar/helper';
-import { useLocation } from 'react-router-dom';
+import { logOut } from '../../store/user/slice';
 import { useMemo } from 'react';
+import { user } from '../../store/user/selectors';
+import { Dropdown, Menu, MenuItem } from '@mui/base';
 import {
   HeaderContainer,
   IconsContainer,
-  PhotoContainer,
+  Listbox,
   ProfileContainer,
+  StyledButton,
   StyledHeaderText,
   StyledHr,
+  StyledMenuButton,
   StyledTypography,
 } from './styled';
+import { useDispatch, useSelector } from 'react-redux';
+import { useLocation, useNavigate } from 'react-router-dom';
 
 const Header: React.FC = () => {
+  const currentUser = useSelector(user);
```

Comment:
look at the prev comments, please use currentUser or user instead of userStore

Reply:
currentUser is used